src/rust/util/src/bb02_async.rs

```diff
@@ -51,7 +51,6 @@ pub fn option<O>(option: &RefCell<Option<O>>) -> AsyncOption<'_, O> {
 }
 
 /// Polls a future until the result is available.
-#[cfg(feature = "testing")]
```

Comment:
I don't see why it is needed in non-testing builds again.

Reply:
Rebase error, fixed.